docs/content/reference/versioning-support.md

```diff
@@ -14,6 +14,18 @@ There are different components that might require independent versioning and sup
 - A HO is intended to be released for each OCP release.
 - A HO released for OCP N minor release must support N, N-1 and N-2 OCP minor releases.
 - A HO release if anything, will only do best effort to support future OCP versions.
+- The HO, once installed, it will create a ConfigMap called `supported-versions` into the Hypershift namespace which describes the HostedClusters supported versions that could be deployed. This is how looks like:
```

Comment:
s/once installed, it will/once installed, will/
s/This is how looks like/This is how it looks like/

minor nits

Reply:
You got it :)